### PR TITLE
Add `review_date` front matter documentation

### PR DESCRIPTION
**Pull request by jdbaldry**

### CODE REVIEW

File: docs/sources/write/front-matter/index.md [hunk 1]
```diff
@@ -298,6 +298,16 @@ Use the `refs` front matter with `ref` URIs to link to different pages in reused
 
 For more information, refer to [Link from source content that’s used in multiple projects](https://grafana.com/docs/writers-toolkit/write/links/#link-from-source-content-thats-used-in-multiple-projects).
 
+### Review date
+
+Use the `review_date` front matter to set the time you last reviewed a page for correctness.
```
Comment (chri2547) [inline docs/sources/write/front-matter/index.md:303]:
```suggestion
Use the `review_date` front matter to set the date you last reviewed a page for correctness.
```

Comment (chri2547) [reply docs/sources/write/front-matter/index.md:303]: Changing this to "date" because time is more specific, ie 2:13pm.

**Review by chri2547 — COMMENTED** [review]
one nit, thanks!

File: docs/sources/write/front-matter/index.md (hunk 1 above)
Comment (jdbaldry) [reply docs/sources/write/front-matter/index.md]: Good point! I think you can actually set a very specific time but I omitted that for simplicity and given that omission, it's more correct to talk about dates and not times.

**Review by chri2547 — APPROVED** [review]

**Review by clayton-cornell — APPROVED** [review]

**Pull request merged**